crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -457,6 +460,12 @@ pub struct AuthorityEpochTables {
     // the accumulator is complete wrt the checkpoint
     pub state_hash_by_checkpoint: DBMap<CheckpointSequenceNumber, Accumulator>,
 
+    /// Maps checkpoint sequence number to the running (non-finalized) root state
+    /// accumulator up th that checkpoint. This should be equivalent to the root
+    /// state hash at end of epoch. Guaranteed to be written to in checkpoint
```

Comment:
this comment probably needs a little updating

Reply:
what part of it do you see as incorrect? This is the updated comment

Reply:
> This should be equivalent to the root state hash at end of epoch

its not clear what "this" refers to here, and I assumed it was from the previous design. Would be clearer to say that each entry is the state hash as of that entry's checkpoint #.